common-lisp-parachute/tests/tests.lisp

```diff
@@ -0,0 +1,12 @@
+(in-package :gilded-rose-tests)
+
+(define-test gilded-rose-testsuite)
+
+(define-test "An item in gilded-rose is updated."
+  :parent gilded-rose-testsuite
+  (let* ((an-item (make-instance 'item :name "An item" :sell-in 1 :quality 1))
```

Comment:
Could you make the values the same as the other languages? ie the item name as "foo" and sell-in and quality both 0